datalad/distribution/create_sibling.py

```diff
@@ -336,23 +346,22 @@ def __call__(sshurl, target=None, target_dir=None,
                 lgr.error("Failed to run post-update hook under path %s. "
                           "Error: %s" % (path, exc_str(e)))
 
-        if target:
-            # add the sibling(s):
-            lgr.debug("Adding the siblings")
-            if target_url is None:
-                target_url = sshurl
-            if target_pushurl is None and sshurl != target_url:
-                target_pushurl = sshurl
-            AddSibling()(dataset=ds,
-                         name=target,
-                         url=target_url,
-                         pushurl=target_pushurl,
-                         recursive=recursive,
-                         fetch=True,
-                         force=existing in {'replace'},
-                         as_common_datasrc=as_common_datasrc,
-                         publish_by_default=publish_by_default,
-                         publish_depends=publish_depends)
+        # add the sibling(s):
+        lgr.debug("Adding the siblings")
+        if target_url is None:
+            target_url = sshurl
+        if target_pushurl is None and sshurl != target_url:
+            target_pushurl = sshurl
+        AddSibling()(dataset=ds,
+                     name=sibling_name,
```

Comment:
Here you go - why but just a `name` since the command is ivory about a sibling? And then actually uniform with add-sibling ;-)